contacts/html-helper.js

```diff
@@ -0,0 +1,229 @@
+  const image = document.createElement(TAGS.IMAGE);
+  image.src = person.image;
+  image.alt = ALT.PIC;
+  image.className = CLASSES.AVATAR;
+  parent.appendChild(image);
+};
+
+const createCheckboxPersonHTML = (parent, person) => {
+  const checkbox = document.createElement(TAGS.INPUT);
+  checkbox.type = TYPES.CHECKBOX;
+  checkbox.custom = CUSTOMS.CHECKBOX;
+  checkbox.id = person.checkBoxId;
+  checkbox.disabled = false;
+  checkbox.checked = person.isChecked;
+  parent.appendChild(checkbox);
+};
+
+const createFullNamePersonHTML = (parent, person) => {
+  const fullName = document.createElement(TAGS.SPAN);
+  fullName.id = person.fullNameId;
+  fullName.className = CLASSES.FULL_NAME;
+  const firstName = document.createElement(TAGS.SPAN);
+  firstName.innerHTML = person.firstName;
+  firstName.className = CLASSES.MARGIN_RIGHT;
+  fullName.appendChild(firstName);
+  const lastName = document.createElement(TAGS.SPAN);
+  lastName.innerHTML = person.lastName;
+  fullName.appendChild(lastName);
+  parent.appendChild(fullName);
+};
+
+const createTrashPersonHTML = (parent, person) => {
+  const trash = document.createElement(TAGS.IMAGE);
+  trash.id = person.trashId;
+  trash.src = IMAGES.TRASH;
+  trash.alt = ALT.TRASH;
+  trash.custom = CUSTOMS.TRASH;
+  trash.className = CLASSES.TRASH;
+  parent.appendChild(trash);
+};
+
+const addToHTMLNewPerson = (person) => {
+  const newLi = createListItemPersonHTML(getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST), person);
+  createPlusButtonPersonHTML(newLi, person);
+  createCountLikesPersonHTML(newLi, person);
+  createMinusButtonPersonHTML(newLi, person);
+  createAvatarImagePersonHTML(newLi, person);
+  createCheckboxPersonHTML(newLi, person);
+  createFullNamePersonHTML(newLi, person);
+  createTrashPersonHTML(newLi, person);
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST).appendChild(newLi);
+};
+
+
+export const populateContactsListHTML = () => {
+  ContactsService.internalArrayContacts.forEach(element => addToHTMLNewPerson(element));
+};
+
+
+const reDrawContactsList = (person = 0, target = 0) => {
+  switch (target) {
+    case CUSTOMS.MINUS_BUTTON:
+      document.getElementById(person.likesCountId).innerHTML = person.likes;
+      document.getElementById(person.minusButtonId).disabled = isCountEqualToMin(person);
+      document.getElementById(person.plusButtonId).disabled = isCountEqualToMax(person);
+      break;
+    case CUSTOMS.PLUS_BUTTON:
+      document.getElementById(person.likesCountId).innerHTML = person.likes;
+      document.getElementById(person.minusButtonId).disabled = isCountEqualToMin(person);
+      document.getElementById(person.plusButtonId).disabled = isCountEqualToMax(person);
+      break;
+    case CUSTOMS.TRASH:
+      document.getElementById(person.listItemId).remove();
+      break;
+    default:
+      break;
+  }
+};
+
+
+export const reRenderContactsListHTML = (person = 0, target = 0) => {
```

Comment:
better set the default value for them to null, otherwise later on you might think, just by looking into code and seeing numbers, that they refer to id's

so, you can have it like this:
export const reRenderContactsListHTML = (person = null, target = null) => {
if (!person || !target) {...}